.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Release
+on:
```

Comment:
So this action runs every time a new tag which starts with lower case 'v' is pushed? Is this a good idea? Would it not be better to tie this to the master branch?

Reply:
Hi! It is just a common way to bind releases to tags. And using `v*` prefix is common too.

I think its a bit better than when the release is triggered by any push to master, because some purely technical pushes to master might exist, when release is unwanted.

Reply:
Can you give me an example of such case which rebuilding the jar would be undesirable? We are talking about building and packaging and distribution archive automatically, not about publishing a new release with a new version number.

Would it be possible that the same release would be built again with the same number? So for example if there is a release 1.0.1 and it is built and packaged already, and then another push to master without version bumping, will override the existing 1.0.1 release with some new changes. Which would mean that if people downloaded the old 1.0.1 and then they downloaded the new 1.0.1, they would actually have two different revisions which are named exactly the same. Is this the issue you're talking about?